Applications/Utils/ModelPreparation/PartitionMesh/NodeWiseMeshPartitioner.cpp

```diff
@@ -518,29 +519,6 @@ void NodeWiseMeshPartitioner::writeASCII(const std::string& file_name_base)
     writeNodesASCII(file_name_base);
 }
 
-void NodeWiseMeshPartitioner::resetGlobalNodeIndices()
-{
-    for (std::size_t i = 0; i < _mesh->_nodes.size(); i++)
-    {
-        _mesh->_nodes[i]->setID(_nodes_global_ids[i]);
```

Comment:
If you removed these two members, please move Node::setID() to the private section as it was originally there before.